packages/cxl-lumo-styles/scss/global_adeft_setup.scss

```diff
@@ -0,0 +1,68 @@
+/* Brand */
```

Comment:
I'm not sure I understand where the idea for this CSS set came from.

We're not going to be overriding current design systems fonts or anything else.

Existing design screens may or may not be updated for CXL UI font looks, it doesn't matter - appearance will stay as current design system state designates.

Reply:
What actually this _"appearance will stay as current design system state designates"_ refers to exactly? 
If we keep original theme styling mostly it looks like this (screen). 
Is it OK?
![Screenshot from 2021-01-25 13-04-49](https://user-images.githubusercontent.com/54673942/105703977-55499a80-5f0e-11eb-97b7-58ae45d15f53.png)

Reply:
Yep, it is OK in this beginning phase.

This is exactly what we have to learn - what existing components should get what upgrades.